mudlet-mapper.xml

```diff
@@ -7300,46 +7323,11 @@ function mmp.addWingsAchaea()
   end
   --universe tarot
   if mmp.settings.universe then
-    if mmp.oncontinent(area, "Main") then
-      local tarotLocations =
-        {
-          azdun = 1772,
-          blackrock = 10573,
-          bitterfork = 25093,
-          genji = 10091,
-          manusha = 8730,
-          newthera = 20386,
-          caerwitrin = 17678,
-          shastaan = 2855,
-          mannaseh = 1745,
-          manara = 9124,
-          brasslantern = 30383,
-          mhojave = 39103,
-          thraasi = 35703,
-          newhope = 25581,
-        }
-      for village, roomnum in pairs(tarotLocations) do
-        mmp.tempSpecialExit(
-          roomnum,
-          [=[script:
-          mmp.customwalkdelay(4.5)
-          send("fling universe at ground",false)
-          tempTimer(4,[[send("touch ]=] ..
-          village ..
-          [=[",false)]])
-          ]=],
-          10
-        )
-      end
-    elseif mmp.oncontinent(area, "Meropis") then
-      local tarotLocations =
-        {
-          rageteeth = 1226,
-          seleucar  = 29755,
-          judgement = 35450,
-          yggdrasil = 12207,
-        }
-      for village, roomnum in pairs(tarotLocations) do
+    local continent = mmp.getareacontinents(area)[1]
```

Comment:
I really don't like this one, but I figured the way mmp.oncontinent is implemented, at some point someone agreed for Achaea special exits to not care for multiple continent areas.  Or maybe I'm missing something

Reply:
I haven't looked at other continent related handling of other special exits, but we could probably loop this:

```lua
local continents = mmp.getareacontinents(area)
for _, continent in ipairs(continents) do
  local universeExits = universeLocations[continent]
  -- the rest of the function
end
```